src/System.Numerics.Vectors/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard1.0;
+      net45;
```

Comment:
do we need a net45 and net46?

Reply:
Yes, in net45 we need a versioned reference that's all typedefs.  In net46 we need a versioned reference that has some typeforwards.